### PR TITLE
[#IA] Medicine tests

### PR DESCRIPTION
**Pull request by maja-tepavcevic**

### CODE REVIEW

File: IntegrationAdapters/Controllers/MedicineController.cs [hunk 1]
```diff
@@ -115,6 +115,11 @@ public IActionResult GetActionsAndBenefits()
             return Ok(Program.ResponseMessageGrpc);
         }
 
-        
-    }
-}
+        [HttpGet("getMedicine")]
+        public IActionResult GetMedicine()
+        {
+            medicineService.AddMedicine();
```
Comment (milicasiriski) [inline IntegrationAdapters/Controllers/MedicineController.cs]: Is this a mistake?

**Review by milicasiriski — CHANGES_REQUESTED** [review]
All looks good, just that the service method Add is called from controller method Get which I think is some oversight

File: IntegrationAdapters/Controllers/MedicineController.cs (hunk 1 above)
Comment (maja-tepavcevic) [reply IntegrationAdapters/Controllers/MedicineController.cs]: Yes, it is mistake. It was add only for adding data in our local database. We will delete it. Thanks

Comment (milicasiriski) [reply IntegrationAdapters/Controllers/MedicineController.cs]: No problem 😄

**Review by milicasiriski — APPROVED** [review]

**Review by VladimirBudjen — APPROVED** [review]

**Pull request merged**